Packs/StellarCyber/Integrations/StellarCyber/test_data/outputs/get_alert.json

```diff
@@ -0,0 +1,241 @@
+{
+    "took": 6,
+    "timed_out": false,
+    "_shards": {
+        "total": 2,
+        "successful": 2,
+        "skipped": 0,
+        "failed": 0
+    },
+    "hits": {
+        "total": {
+            "value": 1,
+            "relation": "eq"
+        },
+        "max_score": 1.0,
+        "hits": [
+            {
+                "_index": "stellar-index-v1-ser-5593fbd8b0444b1eaef5a89589d788d2-64486c346020c889507f32ae-2024.03.07-000033",
+                "_type": "_doc",
+                "_id": "1710883791406342b1f41b2247774d60bf035a6f98e5ff21",
+                "_score": 1.0,
+                "_source": {
+                    "aella_tuples": "131.186.44.94.null.null.null",
+                    "anomaly_id": "c1udWI4BgU-RVXJ6pL6v",
+                    "appid_stdport": "yes",
+                    "detected_fields": [
+                        "email.from.address",
+                        "email.sender.address"
+                    ],
+                    "detected_values": [
+                        "srso+yf09=cw=testing.com=jane@testing.com",
+                        "srso+yf09=cw=testing.com=jane@testing.com"
```

Comment:
Customer names, emails, crypto addresses, email content etc etc that aren't dummy ([example.com](http://example.com/), [demisto.com](http://demisto.com/) for example) shouldn't be exposed as test data, hardcoded or in any way.

If jane@testing.com is not sensitive information, please add it to secret ignore.